src/main/scala/mesosphere/chaos/http/HttpConf.scala

```diff
@@ -17,18 +17,28 @@ trait HttpConf extends ScallopConf {
     descr = "The port to listen on for HTTPS requests", default = Some(8443),
     noshort = true)
 
-  lazy val sslKeystorePath = opt[String]("ssl_keystore_path",
+  lazy val sslKeystorePath = opt[String](
+    "ssl_keystore_path",
     descr = "Path to the keystore, if supplied, SSL is enabled",
-    default = None, noshort = true)
-
-  lazy val sslKeystorePassword = opt[String]("ssl_keystore_password",
```

Comment:
Could we make this more upgrade-friendly?  For example, we could accept the option but then emit a deprecation / misconfiguration message.